app/services/irs_attempts_api/tracker.rb

```diff
@@ -41,15 +41,11 @@ def track_event(event_type, metadata = {})
         event_metadata: event_metadata,
       )
 
-      if IdentityConfig.store.irs_attempt_api_payload_size_logging_enabled
+      if enabled?
         analytics.irs_attempts_api_event_metadata(
           event_type: event_type,
-          unencrypted_payload_num_bytes: event.payload.to_json.bytesize,
```

Comment:
I do think we use the bytes in our dashboards currently, can we keep this? maybe we can memoize the payload generation so we don't double-create?